service/src/integrationTest/kotlin/fi/espoo/evaka/attachments/AttachmentsControllerIntegrationTest.kt

```diff
@@ -0,0 +1,66 @@
+package fi.espoo.evaka.attachments
+
+import com.github.kittinunf.fuel.core.FileDataPart
+import com.github.kittinunf.fuel.core.isSuccessful
+import fi.espoo.evaka.FullApplicationTest
+import fi.espoo.evaka.application.ApplicationStatus
+import fi.espoo.evaka.application.deleteApplication
+import fi.espoo.evaka.insertApplication
+import fi.espoo.evaka.insertGeneralTestFixtures
+import fi.espoo.evaka.resetDatabase
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.auth.UserRole
+import fi.espoo.evaka.shared.auth.asUser
+import fi.espoo.evaka.shared.db.handle
+import fi.espoo.evaka.testAdult_5
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.io.File
+import java.util.UUID
+
+class AttachmentsControllerIntegrationTest : FullApplicationTest() {
+
+    private lateinit var user: AuthenticatedUser
+    private val applicationId = UUID.randomUUID()
+    private val pngFile = this::class.java.getResource("/attachments-fixtures/espoo-logo.png")
+
+    @BeforeEach
+    protected fun beforeEach() {
+        jdbi.handle { h ->
```

Comment:
New code shouldn't use JDBI directly anymore. All the test superclasses expose a `Database.Connection` instance, so you can use `db.transaction { ... }` here